.circleci/config.yml

```diff
@@ -166,6 +166,7 @@ jobs:
           name: Tests real browser
           environment:
             HTTPBIN_BASEURL: 'http://localhost/'
+            TEST_ENV_VAR: 'foo'
```

Comment:
👍 but I'd keep it local like in https://github.com/mui/mui-toolpad/pull/2103/files#diff-f16a8b4fc36e8dc38f1edc79e02791f23fed2ae1be7d7f7b1b015c1a05aec6ebR23-R25

Reply:
Ah so there is a way, I was looking into this right now, thanks!